koji_wrapper/util.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
+""" Collection of utility methods  """
+
+def convert_to_list(value):
+    if isinstance(value, (str, int)):
```

Comment:
You may want to do a list check and if its not already one make a list. I know that is negitive assertion, but a boolean/float/etc would not be returned as a list.

Reply:
While I agree this would make the method more flexible and robust, where we use it now (setters for blacklist and nvr_blacklist), we require the user to only pass in a string or list.  I would propose we add the ability to handle other types as an enhancement if and when we have the need to convert them.  In fact, maybe I shouldn't have even included int here, as it will never get called currently.  What do you think?

Reply:
I would say, lets only handle strings or lists and raise a type error if we get something else. Then we can easily expend later if we choose.